snippets/collection-navigation.liquid

```diff
@@ -0,0 +1,90 @@
+{{ "collection-navigation.css" | asset_url | stylesheet_tag }}
+
+{% if mode == "mobile" %}
```

Comment:
I talked with @hrmndb about this one.
It would be great if we could move the rigger for the sidebar toggle into the top menu as a hamburger icon. This would make it much more clear and useable.
Since we don't support normal menus in the simple store this should be possible. I assume we'll have to rewrite the CSS-only menu triggering into some old-school javascript to achieve this.
<img width="1338" height="1829" alt="image (5)" src="https://github.com/user-attachments/assets/0a24bec3-10f1-48bd-99e5-06fbe7e87f74" />